tests/util_support/test_metadata_file.py

```diff
@@ -7,7 +9,7 @@
 def prepare_file(tmpdir):
     def _prepare_file(contents):
         with open(f"{tmpdir}/metadata.txt", "w") as file:
-            file.write(contents)
+            file.write(re.sub("^\s*", "", contents))
```

Comment:
The regular expression pattern string here needs to be a raw string (`r"^\s*"`) to avoid deprecation warnings from Python 3.6 and later. Here is the warning from the test runner:

```
tests/util_support/test_metadata_file.py:12
  /Users/jlhudd/projects/nextstrain/pulls/augur/tests/util_support/test_metadata_file.py:12: DeprecationWarning: invalid escape sequence \s
    file.write(re.sub("^\s*", "", contents))
```

Reply:
Changed.